domain-models-runtime/src/main/java/org/folio/rest/RestVerticle.java

```diff
@@ -759,6 +759,17 @@ private boolean isStreamed(Annotation[] annotations) {
     return false;
   }
 
+  static Response getResponse(AsyncResult<Response> asyncResult) {
```

Comment:
How does this function relate to the result from the `postTenant` method, would this method be used to interpret it?

Reply:
Yes, I've added javadoc explaining this. This method is package-private.

Reply:
It is used in one place only, in the first line of sendResponse:
`Response responseFromResult = getResponse(v);`